# HG changeset patch
# User Rik <rik@octave.org>
# Date 1518723716 28800
#      Thu Feb 15 11:41:56 2018 -0800
# Branch stable
# Node ID 36ca11f785539dcec5e9c086c8a5037cbc61f19b
# Parent  10e8dc9d453c124b8e3ba153edc64d8845ebc176
Document second calling form of disp().

* pr-output.cc (Fdisp): Document second calling form where string is returned.

diff --git a/libinterp/corefcn/pr-output.cc b/libinterp/corefcn/pr-output.cc
--- a/libinterp/corefcn/pr-output.cc
+++ b/libinterp/corefcn/pr-output.cc
@@ -3479,17 +3479,18 @@ If the length of the smallest possible r
         }
     }
 
   return ovl (string_vector (lst));
 }
 
 DEFUN (disp, args, nargout,
        doc: /* -*- texinfo -*-
-@deftypefn {} {} disp (@var{x})
+@deftypefn  {} {} disp (@var{x})
+@deftypefnx {} {@var{str} =} disp (@var{x})
 Display the value of @var{x}.
 
 For example:
 
 @example
 @group
 disp ("The value of pi is:"), disp (pi)
 
